datadog_checks_dev/datadog_checks/dev/tooling/signing.py

```diff
@@ -107,7 +105,8 @@ def update_link_metadata(checks, core_workflow=True):
         key_id_prefix = key_id[:8].lower()
 
         tag_link = f'{STEP_NAME}.{key_id_prefix}.link'
-        options = {'gpg_keyid': key_id}
+        # Normalize line endings, particularly on Windows, to deal with git.
+        options = {'gpg_keyid': key_id, 'normalize_line_endings': True}
```

Comment:
1. I'm not sure to understand why `normalize_line_endings` being true/false depend on `core_workflow` condition (IIUC that's wether or not we are using `core` repo).

In case it's for windows only, why not check for the platform condition instead?

2. 

> That works for developers on Windows (where they must hash files that end up being normalized in git), but not for extras on Linux.

What makes it not work for extras on linux ?